DESCRIPTION

```diff
@@ -46,8 +46,6 @@ Suggests:
     scda.2021 (>= 0.1.3),
     teal.data (>= 0.1.1),
     testthat (>= 2.0)
-VignetteBuilder:
```

Comment:
https://github.com/insightsengineering/teal.osprey/issues/146#issuecomment-1241805766  --as-cran complains there are no vignettes but vignettebuilder is specified